src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumesRepository.cs

```diff
@@ -78,23 +78,51 @@ public class ResumesRepository : IResumesRepository
                     CreationDate = DateTime.Now.AddDays(-10),
                     LastUpdate = DateTime.Now
                 }
-            };
+            }.ToList();
 
-        public Resume GetResume(Guid id)
+        public Resume Add(Resume newObject)
         {
-            Resume result = resumes.FirstOrDefault(p => Equals(p.Id, id));
-            return result;
+            Resumes.Add(newObject);
+            return Resumes.Last();
+        }
+
+        public void Delete(Guid id)
+        {
+            throw new NotImplementedException();
+        }
+
+        public List<Resume> GetAll()
+        {
+            return Resumes.ToList();
+        }
+
+        public Resume GetById(Guid id)
+        {
+            foreach (Resume item in Resumes)
+            {
+                if (item.Id == id)
+                {
+                    return item;
+                }
+            }
+
+            return null;
         }
 
         public IEnumerable<Resume> GetResumes()
         {
-            return resumes;
+            return Resumes;
         }
 
         public Resume PostResumes(Resume resume)
         {
-            resumes = new List<Resume>(resumes) { resume }.ToArray();
+            var resumes = new List<Resume>(Resumes) { resume }.ToArray();
             return resume;
         }
```

Comment:
You have to remove these methods

Reply:
And change "IResumesRepository" to "IRepository<Resume>" in core tests, after this LGTM

Reply:
These methods are implementing the Resume swagger endpoints.
Changes made.

Reply:
Make the change in PostResumesCoreTests.cs too

Reply:
The new methods add and getAll should work too, what do you say @cristian-chavez-jalafund

Reply:
Changes made.
Add methods.